src/00_hello.py

```diff
@@ -1 +1,2 @@
-# Print "Hello, world!" to your terminal
\ No newline at end of file
+# Print "Hello, world!" to your terminal
+print ('Hello there')
```

Comment:
Definitely not a huge deal in this case, but generally try to match the expected output you're given exactly